src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure implements FindArea {
+    private int radius;
+
+    public Circle() {
+        super();
+        this.radius = new Random().nextInt(1, 10);
+    }
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return 2 * Math.PI * Math.pow(radius, 2);
```

Comment:
The formula for the area of a circle is incorrect. It should be `Math.PI * Math.pow(radius, 2)` instead of `2 * Math.PI * Math.pow(radius, 2)`. The current formula calculates the circumference, not the area.